### DISCUSSION TITLE
Is it Possible to use  RVC (Retrieval-based-Voice-Conversion-WebUI) with Coqui TTS by using .pth and .index files?

### DISCUSSION THREAD

**Discussion by Rajashekhar-Reddy** [coqui-ai/TTS#2820] [General]
If Coqui TTS doesn't have that ability, is there any other Open Source TTS that has the ability to use RVC models with TTS?

Anyone has any idea about  other TTS projects which has the ability to use RVC-trained files (.pth, .index) for Voice TTS

**Comment by erogol** [comment]
If the goal is voice conversion, it is possible with TTS but if it is only RVC, yes TTS does not support RVC.

**Comment by miranda1000** [comment]
> Anyone has any idea about other TTS projects which has the ability to use RVC-trained files (.pth, .index) for Voice TTS

Check [rvc-tts-webui](https://github.com/litagin02/rvc-tts-webui).
